src/Utilities/universalfallback.jl

```diff
@@ -76,6 +76,24 @@ function MOI.get(uf::UniversalFallback, attr::Union{MOI.AbstractVariableAttribut
     end
 end
 
+_dict(uf, attr::MOI.ListOfOptimizerAttributesSet)  = uf.optattr
```

Comment:
I don't find the `_dict`/`_args` indirection very helpful. Someone is going to be unnecessarily confused when reading this code. (E.g.Why are we splatting empty tuples?) I would just a standalone implementation for each of the ListOfXXXSet attributes.